packages/docs/src/routes/docs/(qwikcity)/guides/qwik-nutshell/index.mdx

```diff
@@ -180,6 +180,27 @@ export default component$(() => {
 
 Remember that Qwik uses `class` instead of `className` for CSS classes.
 
+```tsx title="src/components/MyComponent/MyComponent.tsx"
```

Comment:
I am not sure this should be added here, in the CSS Modules section.

Reply:
I have moved the example to the beginning of the document, just below where it mentions not using classname. 
What do you think?